File: src/test/java/backend/TestBackendConfiguration.java
package backend;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(exclude = {SwaggerConfiguration.class})
public class TestBackendConfiguration {

    public static void main(String[] args) {
        SpringApplication.run(BackendConfiguration.class, args);
    }

}

File: src/test/java/backend/TestBackendConfiguration.kt
@file:JvmName("SpringBootApplication")

package backend

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching

@SpringBootApplication
@EnableCaching
open class TestBackendConfiguration {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(Application::class.java, *args)
        }
    }
}
